sdcm/fill_db_data.py

```diff
@@ -3052,9 +3052,9 @@ def insert_query():
 
     @staticmethod
     def cql_truncate_simple_tables(session, rows):
-        truncate_query = 'TRUNCATE TABLE truncate_table%d'
+        truncate_query = 'TRUNCATE TABLE truncate_table%d USING TIMEOUT 600'
         for i in range(rows):
-            session.execute(truncate_query % i)
+            session.execute(truncate_query % i, timeout=600)
```

Comment:
the timeout was:
```
session.default_timeout = 60.0 * 5
```

how's repeating it in 3 different places gonna help ? 

also in lot of the cases truncate is failing, and not timing out, hence it won't cover all of the issue with truncate